google-cloud-storage/src/main/java/com/google/cloud/storage/ApiaryConversions.java

```diff
@@ -472,6 +477,22 @@ private IamConfiguration iamConfigDecode(Bucket.IamConfiguration from) {
     return to.build();
   }
 
+  private Bucket.Autoclass autoclassEncode(Autoclass from) {
+    Bucket.Autoclass to = new Bucket.Autoclass();
+    if(from.getEnabled() != null) {
+      to.setEnabled(from.getEnabled());
+    }
```

Comment:
nit: this can use ifNonNull as well
```suggestion
    ifNonNull(from.getEnabled(), to::setEnabled);
```